### DISCUSSION TITLE
Portainer issue reveals podman problems

### DISCUSSION THREAD

**Discussion by yajo** [containers/podman#10349] [General]
It seems Portainer is considering support for podman, and somebody from their community found out a few rough edges between its API and Docker's: https://github.com/portainer/portainer/issues/2991#issuecomment-805805031

I just wanted to point it out here, because that's probably value info for podman maintainers. After all, portainer is a very docker-specific manager that, if made to work, would definitely prove the API compatibility level.

**Comment by mheon** [comment]
We encourage them to open issues on anything they find, including all the information in the template.

I will admit that a number of the issues they're raising confuse me - they talk about a `containers/{id}/recreate` API endpoint? Docker does not have such a thing. Neither do we. Perhaps they're talking about `restart`?

**Reply by yajo** [reply]
Yes, I found that confusing too. 🤷‍♂️
